floss/render/default.py

```diff
@@ -301,7 +301,7 @@ def get_color(color):
 
 def render(results: floss.results.ResultDocument, verbose, disable_headers, color):
     sys.__stdout__.reconfigure(encoding="utf-8")
-    console = Console(file=io.StringIO(), color_system=get_color(color), highlight=False)
+    console = Console(file=io.StringIO(), color_system=get_color(color), highlight=False, redirect=False)
```

Comment:
should this be using `soft_wrap=True`? i don't see how this change affects the wrapping.

Reply:
So we could also achieve this with `rich` and setting `soft_wrap=True`?